wcomponents-core/src/main/java/com/github/bordertech/wcomponents/AbstractInput.java

```diff
@@ -303,17 +303,13 @@ protected void doHandleChanged() {
 		if (getActionOnChange() != null) {
 			final ActionEvent event = new ActionEvent(this, getActionCommand(), getActionObject());
 			final boolean isCAT = isCurrentAjaxTrigger();
-			Runnable later = new Runnable() {
-				@Override
-				public void run() {
-					getActionOnChange().execute(event);
-					if (isCAT && UIContextHolder.getCurrent().getFocussed() == null) {
-						setFocussed();
-					}
-				}
-			};
 
-			invokeLater(later);
+			invokeLater(() -> {
+				getActionOnChange().execute(event);
+				if (isCAT && UIContextHolder.getCurrent().getFocussed() == null) {
+					setFocussed();
+				}
+			});
```

Comment:
This makes sense.

A slightly smaller change would have been to leave the variable assignment like so:

```java
Runnable later = () -> {
	getActionOnChange().execute(event);
	if (isCAT && UIContextHolder.getCurrent().getFocussed() == null) {
		setFocussed();
	}
};

invokeLater(later);
```

It's no big deal tho, I don't see any major advantage or disadvantage in this case.